packages/palette/src/elements/Disclosure/Disclosure.story.tsx

```diff
@@ -0,0 +1,55 @@
+import { action } from "@storybook/addon-actions"
+import React from "react"
+import { States } from "storybook-states"
+import { Clickable } from "../Clickable"
+import { Flex } from "../Flex"
+import { Text } from "../Text"
+import { Disclosure, DisclosureProps } from "./Disclosure"
+import { Button } from "../Button"
+
+export default {
+  title: "Components/Disclosure",
+}
+
+export const Default = () => {
+  return (
+    <States<Partial<DisclosureProps>>
+      states={[
+        {},
+        { expanded: true },
+        {
+          label: (
+            <Flex flex={1} justifyContent="space-between">
+              <Text variant="sm-display">Heading</Text>
+              <Clickable
+                textDecoration="underline"
+                onClick={(e) => {
+                  e.stopPropagation()
+                  action("Secondary action")
+                }}
+              >
+                <Text variant="sm-display">Reset</Text>
+              </Clickable>
```

Comment:
This sort of stuff isn't relevant for this component. I wouldn't simply copy expandable's stories here.

Reply:
I removed a few, this case got me if would make sense to be here or not, going to remove it then